src/workspace-data/data-table/wds/WDSContent.js

```diff
@@ -18,19 +17,21 @@ const WDSContent = ({
   const [refreshKey] = useState(0);
 
   // Render
-  const [entityMetadata, setEntityMetadata] = useState(() => wdsToEntityServiceMetadata(wdsSchema));
+  const [entityMetadata, setEntityMetadata] = useState(() => wdsSchema);
 
+  const id = workspace.workspace.workspaceId;
+  // dataProvider contains the proxyUrl for an instance of WDS
   return h(Fragment, [
     h(DataTable, {
       dataProvider,
       persist: true,
       refreshKey,
-      editable: false,
+      editable: true,
       entityType: recordType,
       activeCrossTableTextFilter: false,
       entityMetadata,
       googleProject,
-      workspaceId: { namespace, name },
+      workspaceId: { namespace, name, id },
```

Comment:
I kind of want to rename this, but I am a bit confused how this is different from the "workspace" variable below it